README.md

```diff
@@ -17,45 +24,42 @@ otCore.init(options) =>
   Annotation +
   Archiving
 ```
+##Using accelerator core
 
-OpenTok Accelerator Core provides a simple way to integrate real-time audio/video into your web application using the OpenTok Platform.  `Core` also integrates with, manages, and provides a single API for the following accelerator packs:
+`OpenTok Accelerator Core` provides a simple way to integrate real-time audio/video into your web application using the OpenTok Platform.  It also integrates with, manages, and provides a single API for the following accelerator packs:
 
  - [Text Chat](https://www.npmjs.com/package/opentok-text-chat)
  - [Screen Sharing](https://www.npmjs.com/package/opentok-screen-sharing)
  - [Annotation](https://www.npmjs.com/package/opentok-annotation)
  - [Archiving](https://www.npmjs.com/package/opentok-archiving)
 
-`Core` also internally maintains the state of your OpenTok session for you.  Calling `otCore.state()` returns an object containing:
-```javascript
-  streams     => All current streams
-  streamMap   => The map of stream ids to publisher/subscriber ids
-  publishers  => All current publishers
-  subscribers => All current subscribers
-  meta        => The count of all current publishers and subscribers by type
-```
-*See an example of the publisher, subscriber, and meta data [below](#pubSubData).*
+More advanced features are also available when using the [Advanced OpenTok Session Management Using SDK-Wrapper](#advanced-opentok-session-management-using-sdk-wrapper). 
```

Comment:
I think that having this here is a little bit confusing, as people may not be sure whether they should be using `core` or the `sdk-wrapper`.  `sdk-wrapper` should be a secondary choice, but I think that we should gently direct people to use `core`.  That's why I originally only mentioned `sdk-wrapper` at the end.  There may be a better place for it though.